### PR TITLE
refactor: main js

### PR DESCRIPTION
**Pull request by setchy**
update to closer match structure of [examples](https://www.electronjs.org/docs/latest/tutorial/examples)

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -1,18 +1,17 @@
-const { ipcMain: ipc, app, nativeTheme } = require('electron');
+const { ipcMain: ipc, app, nativeTheme } = require('electron/main');
 const { menubar } = require('menubar');
 const { autoUpdater } = require('electron-updater');
 const { onFirstRunMaybe } = require('./first-run');
 const path = require('node:path');
 
-require('@electron/remote/main').initialize();
```
Comment (setchy) [inline main.js:7]: i dont think this is required...

**Review by bmulholland — APPROVED** [review]
Huge fan of this change. These types of updates are easy to overlook, but often solve hours of unnecessary debugging. Thanks!

File: main.js (hunk 1 above)
Comment (setchy) [inline main.js:7]: @bmulholland - thoughts on if this is OK to remove now.  I know you were cleaning this up via https://github.com/gitify-app/gitify/pull/654

Comment (afonsojramos) [reply main.js:7]: If it doesn't work it doesn't, but otherwise I see no reason to not remove it!

Comment (setchy) [reply main.js:7]: I don't want to pull the trigger on this yet.  left a TODO referencing #650 for future us

**Pull request merged**